code/game/gamemodes/wizard/artefact.dm

```diff
@@ -213,3 +213,383 @@
 	user << "<span class='notice'> You can see...everything!</span>"
 	visible_message("<span class='danger'>[user] stares into [src], their eyes glazing over.</span>")
 	user.ghostize(1)
+
+/////////////////////Multiverse Blade////////////////////
+var/global/list/multiverse = list()
+
+/obj/item/weapon/multisword
+	name = "multiverse sword"
+	desc = "A weapon capable of conquering the universe and beyond. Activate it to summon copies of yourself from others dimensions to fight by your side."
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "energy_katana"
+	item_state = "energy_katana"
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	flags = CONDUCT
+	slot_flags = SLOT_BELT
+	force = 20
+	throwforce = 10
+	sharp = 1
+	edge = 1
+	w_class = 2
+	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+	var/faction = list("unassigned")
+	var/cooldown = 0
+	var/cooldown_between_uses = 400 //time in deciseconds between uses--default of 40 seconds.
+	var/assigned = "unassigned"
+	var/evil = TRUE
+	var/probability_evil = 30 //what's the probability this sword will be evil when activated?
+	var/duplicate_self = 0 //Do we want the species randomized along with equipment should the user be duplicated in their entirety?
+	var/sword_type = /obj/item/weapon/multisword //type of sword to equip.
+
+/obj/item/weapon/multisword/New()
+	..()
+	multiverse |= src
+
+
+/obj/item/weapon/multisword/Destroy()
+	multiverse.Remove(src)
+	return ..()
+
+/obj/item/weapon/multisword/attack(mob/living/M as mob, mob/living/user as mob)  //to prevent accidental friendly fire or out and out grief.
+	if(M.faction == user.faction)
```

Comment:
Given lists are objects (Set two list vars to the same list reference and they will have the same list and update whenever the other var's ref to it updates) this check only ensures that the list object is the same, and since it's created on New() for each mob, it won't be (Unless fox's code here copies the same list object over, in which case -maybe-)

tl;dr bad check,